rllib/utils/checkpoints.py

```diff
@@ -313,10 +345,21 @@ def restore_from_path(
                 the subcomponent and thus, only that subcomponent's state is
                 restored/loaded. All other state of `self` remains unchanged in this
                 case.
+            filesystem: PyArrow FileSystem to use to access data at the path. If not
```

Comment:
Maybe use backticks around `path` to indicate that we mean the value of the `path` arg.

Reply:
Good catch. Maybe I also turn "PyArrow" to `pyarrow`.